.github/workflows/update-geoip.yml

```diff
@@ -0,0 +1,72 @@
+name: Update GeoIP DB
+
+on:
+  # Run every Monday at 13:37
+  schedule:
+    - cron: '37 13 * * 1'
+  # Run on every push to `master`
+  push:
+    branches: [ master ]
+  # And whenever this workflow is updated or a PR attempts to update the GeoIP files.
+  pull_request:
+    paths:
+      - '.github/workflows/update-geoip.yml'
+      - 'includes/geo/*'
+  # And manually
+  workflow_dispatch:
+
+# Cancels all previous workflow runs for the same branch that have not yet completed.
+concurrency:
+  # The concurrency group contains the workflow name and the branch name.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  certificate-check:
```

Comment:
It could be easier to maintain to have the two update jobs in a single workflow file

Reply:
Once everything works sure :)

Reply:
On second thoughts I'm not so sure about this.
* The 2 workflows work differently and I think I'll have an easier time debugging when one breaks
* The cacert workflow is "safe" : before committing there's a checksum to make sure the file is valid. There is no such thing for the geoip workflow and in the possibility that the download may be corrupted we need to run unit tests on these PR